### DISCUSSION TITLE
Publish Biome on Homebrew

### DISCUSSION THREAD

**Discussion by nhedger** [biomejs/biome#197] [Ideas]
Publishing Biome to Homebrew would make installing it easier for macOS users.

Edit: tracked in https://github.com/biomejs/biome/issues/209

**Comment by ematipico** [comment]
That would be nice. I have no knowledge on how to publish a binary as a homebrew formula though.

It would be great if someone can take a stab at it

**Reply by nhedger** [reply]
I'll take care of it.

**Reply by nhedger** [reply]
Actually, I'll hold off until the crates are renamed.

**Comment by nhedger** [comment]
Biome is now being published on Homebrew.

https://formulae.brew.sh/formula/biome
